From 6d4d132ef591a08127895a5058d580f07124597b Mon Sep 17 00:00:00 2001
From: Mark Hatle <mark.hatle@amd.com>
Date: Wed, 28 Aug 2024 14:35:34 -0600
Subject: [PATCH] openamp: xlnx: Fix typo in comparison

Resolve issue reported as:
 [ERROR]: could not load assist: .../usr/lib/python3.12/site-packages/lopper/assists/baremetal_bspconfig_xlnx.py: expected ':' (openamp_xlnx.py, line 200)
 [ERROR]: could not load assist: .../usr/lib/python3.12/site-packages/lopper/assists/baremetal_bspconfig_xlnx.py: expected ':' (openamp_xlnx.py, line 1037)

Signed-off-by: Mark Hatle <mark.hatle@amd.com>
---
 lopper/assists/openamp_xlnx.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/lopper/assists/openamp_xlnx.py b/lopper/assists/openamp_xlnx.py
index 6fd77b0..5d62000 100644
--- a/lopper/assists/openamp_xlnx.py
+++ b/lopper/assists/openamp_xlnx.py
@@ -197,7 +197,7 @@ def xlnx_rpmsg_construct_carveouts(tree, carveouts, rpmsg_carveouts, native, cha
                 new_node =  LopperNode(-1, "/reserved-memory/"+carveout.name)
                 new_node + LopperProp(name="no-map")
                 new_node + LopperProp(name="reg", value=[0, start, 0, size])
-                if not reserved_mem_node_check(tree, new_node)
+                if not reserved_mem_node_check(tree, new_node):
                     return False
 
                 if "vdev0buffer" in carveout.name:
@@ -1034,7 +1034,7 @@ def xlnx_remoteproc_construct_carveouts(tree, carveouts, new_ddr_nodes, verbose
             new_node + LopperProp(name="reg", value=[0, start, 0, size])
             tree.add(new_node)
 
-            if not reserved_mem_node_check(tree, new_node)
+            if not reserved_mem_node_check(tree, new_node):
                 return False
 
             phandle_val = new_node.phandle_or_create()
-- 
2.34.1
